app/pages/reset-password/new-password-form.spec.js

```diff
@@ -5,15 +5,41 @@ import sinon from 'sinon';
 import NewPasswordForm from './new-password-form';
 
 describe('NewPasswordForm', () => {
+  let wrapper;
+
+  before(() => {
+    wrapper = shallow(<NewPasswordForm />);
+  });
+
   it('renders a new password form for a user', () => {
-    const wrapper = shallow(<NewPasswordForm />);
     assert.equal(wrapper.find('form').length, 1);
   });
 
   it('should call onSubmit when user submits a new password', () => {
     const newPasswordSpy = sinon.spy();
-    const wrapper = mount(<NewPasswordForm onSubmit={newPasswordSpy} />);
+    wrapper = mount(<NewPasswordForm onSubmit={newPasswordSpy} />);
```

Comment:
You shouldn't need to mount this to test a spy. The spy can be setup from the beginning at in a const at the start of the describe block and passed in as a prop in the initial shallow render.